File: Sheet #1 (Data type - Conditions)/C++/R-Age in Days.cpp
#include <iostream>
using namespace std;

int main()
{
    long long int num;
    int years, months, days;

    cin >> num;
    years = num / 365;
    months = (num - (365 * years)) / 30;
    days = (num - (365 * years) - (30 * months));

    cout << years << " years" << endl;
    cout << months << " months" << endl;
    cout << days << " days" << endl;

    return (0);
}

File: Sheet #1 (Data type - Conditions)/python/R-Age in Days.py
#!/usr/bin/python3
"""
Given a Number N corresponding to a person's age (in days).
Print his age in years, months and days,
followed by its respective message
=> "years", "months", "days".
"""

age_in_days = int(input(""))

years = int(age_in_days / 365)

months = int((age_in_days - years * 365) / 30)

days = int(age_in_days - years * 365 - months * 30)

print(f"{years} years")
print(f"{months} months")
print(f"{days} days")
